apps/google-analytics-4/frontend/src/components/main-app/analytics-app/AnalyticsApp.tsx

```diff
@@ -85,15 +91,15 @@ const AnalyticsApp = () => {
           <p className={styles.noteContent}>{error?.message || DEFAULT_ERR_MSG}</p>
         </Note>
       );
-    } else if (!pageViewData.rowCount) {
+    } else if (!runReportResponse.rowCount) {
```

Comment:
I know you didn't do this but in line 91, you don't need the `?` because you're already checking if `error` is undefined/null in line 88